Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories = ["os::windows-apis", "external-ffi-bindings"]
 edition = "2021"
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.52.0"
+version = ">=0.48.0, <=0.59.*"
```

Comment:
OK, so my initial reaction to this was that libraries shouldn't be setting upper bounds. But this looks like it is strictly more flexible than `0.59.0`. Which I think makes sense to me.

Reply:
Yeah. I just wanted to make sure that all sem-ver compatible point releases would be included without also including arbitrary future breaking releases. I could have also used `<0.60.0` but I thought it was a bit clearer to write the current version rather.